packages/firebase_database/README.md

```diff
@@ -9,7 +9,16 @@ For Flutter plugins for other Firebase products, see [README.md](https://github.
 *Note*: This plugin is still under development, and some APIs might not be available yet. [Feedback](https://github.com/FirebaseExtended/flutterfire/issues) and [Pull Requests](https://github.com/FirebaseExtended/flutterfire/pulls) are most welcome!
 
 ## Usage
-To use this plugin, add `firebase_database` as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/).
+To use this plugin, add `firebase_database` as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/). You will also need the `firebase_core` dependency if you do not have it already.
```

Comment:
s/flutter.io/flutter.dev/

Reply:
the url in the diff ultimately leads here: https://flutter.dev/docs/development/packages-and-plugins/using-packages